pkg/summary/options.go

```diff
@@ -0,0 +1,51 @@
+package summary
+
+import (
+	"github.com/fairwindsops/goldilocks/pkg/kube"
+	"github.com/fairwindsops/goldilocks/pkg/utils"
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+// Functional options
```

Comment:
linter: 

```
pkg/summary/options.go:9:1: comment on exported type Option should be of the form "Option ..." (with optional leading article)
```

There's a few of these. I suggest running `golint ./...` on this PR.

Reply:
I've fixed up the golint suggestions.